EMS/common-bundle/src/Command/Index/SynchronizeCommand.php

```diff
@@ -0,0 +1,262 @@
+<?php
+
+declare(strict_types=1);
+
+namespace EMS\CommonBundle\Command\Index;
```

Comment:
Would put this command in the root level, creating an index folder with one command is overkill.